completion_aggregator/settings/common.py

```diff
@@ -52,3 +55,17 @@ def plugin_settings(settings):
     # 1. All courses should be reaggregated for the changes to take effect.
     # 2. It's not possible to revert this change by reaggregation without manually removing existing Aggregators.
     settings.COMPLETION_AGGREGATOR_AGGREGATE_UNRELEASED_BLOCKS = False
+
+    # Whitelist the aggregator events for use with event routing backends xAPI backend.
+    enabled_aggregator_events = [
+        f'openedx.completion_aggregator.{event_type}.{block_type}'
+
+        for event_type in settings.ALLOWED_COMPLETION_AGGREGATOR_EVENT_TYPES
+        for block_type in settings.ALLOWED_COMPLETION_AGGREGATOR_EVENT_TYPES[event_type]
+    ]
+    settings.EVENT_TRACKING_BACKENDS_ALLOWED_XAPI_EVENTS += enabled_aggregator_events
```

Comment:
Ok, this is tricky.. and v. important to get right.

Those settings should be provided by the ERB plugin, which, since it's added as a dependency here, *should* be installed first, and therefore the settings should already have these variables. But more so: if the ERB plugin is loaded after this one, then [ERB will replace these variables with its own settings](https://github.com/openedx/event-routing-backends/pull/431/files#diff-24e49be8ca25acc52569b40114f80c58b30f5025af88fe3a8483da4e6358f6efR90), and we'll lose these events. So the order they load in is critical here.

When you installed this branch on your devstack, did it install my ERB branch from https://github.com/openedx/event-routing-backends/pull/431?

Reply:
Ok, good plan.

I've made your suggested change on ERB: [c5a89fa](https://github.com/openedx/event-routing-backends/pull/431/commits/c5a89fa26917a1b8394b02db178058088a61e1cf), and here 3ff8331fa7ace76a114276781d9391fd700b536c